app/src/main/java/com/dessalines/thumbkey/db/AppDb.kt

```diff
@@ -275,6 +287,22 @@ val MIGRATION_8_9 = object : Migration(8, 9) {
     }
 }
 
+val MIGRATION_9_10 = object : Migration(9, 10) {
+    override fun migrate(database: SupportSQLiteDatabase) {
+        database.execSQL(
+            "alter table AppSettings add column slide_sensitivity INTEGER NOT NULL default $DEFAULT_SLIDE_SENSITIVITY",
```

Comment:
These are both adding the same feature, so you can combine these into a single migration. The execSQL  can have more than one statement.